mindmeld/components/dialogue.py

```diff
@@ -656,6 +658,10 @@ def prompt(self, text):
         self._logger.warning('prompt() is deprecated. Please use reply() and listen() instead')
         self.reply(text)
 
+    def sleep(self):
```

Comment:
Can you add an optional param `delay` with default `0`?

Delay is the number of milliseconds to wait before putting the client to sleep.